.github/pull_request_template.md

```diff
@@ -55,7 +55,7 @@ Uncomment this markdown table below and edit the last line `|||`:
 - [ ] UI change has been tested on both light and dark themes
 - [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
 - [ ] Pull request is based on the develop branch
-- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#changelog
+- [ ] Pull request title will be used in the release note, it clearly define what will change for the user
```

Comment:
Should we mention the usage of label heres too?

Reply:
Danger will complain if the label is missing, so I do not think it worth it. Also I do not think external contributors using forks can add labels (not sure about this).